src/psyclone/transformations.py

```diff
@@ -313,12 +313,171 @@ def apply(self, node, options=None):
         return schedule, keep
 
 
+class OMPTaskloopTrans(ParallelLoopTrans):
+
+    '''
+    Adds an OpenMP taskloop directive to a loop.
+
+    TODO: #1364 Taskloops do not yet support reduction clauses.
+
+    For example:
+
+    >>> from pysclone.parse.algorithm import parse
+    >>> from psyclone.psyGen import PSyFactory
+    >>> api = "gocean1.0"
+    >>> filename = "nemolite2d_alg.f90"
+    >>> ast, invokeInfo = parse(filename, api=api, invoke_name="invoke")
+    >>> psy = PSyFactory(api).create(invokeInfo)
+    >>>
+    >>> from psyclone.transformations import OMPParallelTrans, OMPSingleTrans
+    >>> from psyclone.transformations import OMPTaskloopTrans
+    >>> singletrans = OMPSingleTrans()
+    >>> paralleltrans = OMPParallelTrans()
+    >>> tasklooptrans = OMPTaskloopTrans()
+    >>>
+    >>> schedule = psy.invokes.get('invoke_0').schedule
+    >>> schedule.view()
+    >>>
+    # Apply the OpenMP Taskloop transformation to *every* loop
+    # in the schedule.
+    # This ignores loop dependencies. These must be manually handled
+    # either through end of regions.
+    # TODO: #1368 These can also be handled through the taskwait directive
```

Comment:
Add the ">>> " before the comments and TODO as well, so it looks as the same block in the documentation.

Reply:
Done